arithmetization/src/main/java/net/consensys/linea/zktracer/runtime/callstack/CallFrame.java

```diff
@@ -262,7 +262,11 @@ private void revertChildren(CallStack callStack, int parentRevertStamp) {
             frame -> {
               frame.getsReverted = true;
               if (!frame.selfReverts) {
-                frame.revertStamp = parentRevertStamp;
+                if (frame.revertStamp == 0) {
+                  frame.revertStamp = parentRevertStamp;
+                } else if (frame.revertStamp > parentRevertStamp) {
+                  frame.revertStamp = parentRevertStamp;
+                }
```

Comment:
The `REVERT_STAMP` of a context that `GETS_REVERTED` but doesn't `SELF_REVERT` needs to be that of the nearest reverter context. This is the context with the least `CONTEXT_REVERT_STAMP`

Reply:
The ideas is:
- if not already set, set it
- if a nearer parent self reverted (which will have a smaller `parentRevertStamp`) is their `REVERT_STAMP`